cli.py

```diff
@@ -97,7 +97,7 @@ async def messaging(
                 num_partitions=partitions,
                 replication_factor=replication,
                 topic_retention_time_second=topic_retention_time,
-            )()
+            )
```

Comment:
Also below in line 111 there are double parenthesis, should be removed as well?

Reply:
No, because message factory returns a class. The double set is required below because we need to create an instance of the class where as here `message` is a class so we just need to initialize.

Reply:
> No, because message factory returns a class. The double set is required below because we need to create an instance of the class where as here `message` is a class so we just need to initialize.

Thanks for the explanation :) Then it can be merged for me :)